concept-amendments.html

```diff
@@ -317,12 +324,28 @@ <h2 id="cryptoconditions">CryptoConditions</h2>
 <tbody>
 <tr>
 <td align="left">1562511F573A19AE9BD103B5D6B9E01B3B46805AEC5D3C4805C902B514399146</td>
-<td align="left">In development</td>
+<td align="left">Enabled</td>
 </tr>
 </tbody>
 </table>
-<p>Adds conditional payments that are directly compatible with the <a href="https://tools.ietf.org/html/draft-thomas-crypto-conditions-02">Interledger Protocol Crypto-Conditions specification</a>.</p>
-<p class="devportal-callout caution"><strong>Caution:</strong> This amendment is still in development.</p>
+<p>Although this amendment is enabled, it has no effect unless the <a href="#suspay">SusPay</a> amendment is also enabled. Ripple plans to withdraw SusPay and implement the same functionality as part of the <a href="#escrow">Escrow</a> amendment including crypto-condition support.</p>
+<h2 id="escrow">Escrow</h2>
+<table>
+<thead>
+<tr>
+<th align="left">Amendment ID</th>
+<th align="left">Status</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td align="left">07D43DCE529B15A10827E5E04943B496762F9A88E3268269D69C44BE49E21104</td>
+<td align="left">In Development</td>
+</tr>
+</tbody>
+</table>
+<p>Replaces the <a href="#suspay">SusPay</a> amendment, incorporating functionality from the <a href="#cryptoconditions">CryptoConditions</a> amendment.</p>
+<p>Provides "suspended payments" for XRP for escrow within the Ripple Consensus Ledger including support for <a href="https://tools.ietf.org/html/draft-thomas-crypto-conditions-02">Interledger Protocol Crypto-Conditions</a>. Creates the <code>SuspendedPayment</code> ledger node type and the new transaction types <code>SuspendedPaymentCreate</code>, <code>SuspendedPaymentFinish</code>, and <code>SuspendedPaymentCancel</code>.</p>
```

Comment:
Should there be a command before `including`? Also, we will likely be renaming the code to use different names for the transactions - _potentially_ `EscrowCreate`, `EscrowFinish` and `EscrowCancel` but that's not finalized yet.